public_html/wp-content/plugins/wcpt/wcpt-wordcamp/wordcamp-admin.php

```diff
@@ -761,7 +767,11 @@ public static function notify_new_wordcamp_in_slack( $wordcamp ) {
 
 			$attachment = create_event_status_attachment( $message, $wordcamp->ID, $title );
 
-			return wcpt_slack_notify( COMMUNITY_EVENTS_SLACK, $attachment );
+			$notification_sent = wcpt_slack_notify( COMMUNITY_EVENTS_SLACK, $attachment );
+			if ( $notification_sent ) {
+				update_post_meta( $wordcamp->ID, $scheduled_notification_key, true );
+			}
+			return $notification_sent;
```

Comment:
Same comments as above regarding the `return`, and removing `$already_notified`.

There's a lot of duplicating in these 3 functions. I wonder if we should refactor them to be more DRY? It may not be worth it, though, since it's just a few lines in each.